### DISCUSSION TITLE
Ray Tracing for Lens Design

### DISCUSSION THREAD

**Discussion by mrplants** [idaholab/moose#23602] [Q&A General] [answered]
I'd like to use the ray tracing module to design a system of lenses. Is there a clever way to visualize the rays, themselves, so that I can iterate and focus my design.

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can output the rays to Exodus
https://mooseframework.inl.gov/source/outputs/RayTracingExodus.html

Guillaume

**Reply by mrplants** [reply]
Thanks!
